python/sanlock.c

```diff
@@ -71,6 +71,16 @@ __set_exception(int en, char *msg)
     }
 }
 
+static unsigned long long
+pyinteger_as_ullmask(PyObject *obj)
```

Comment:
I like the short notation, but it doe snot work for long (pyinteger_as_l?), so better keep the long name:

    pyinteger_as_unsigned_long_long_mask

Reply:
ull is commonly used, just like the printf notations. but if you insist.